src/compiler/parsing/watf/number-literals.js

```diff
@@ -29,8 +29,12 @@ export function parse32I( sourceString: string ): number {
   return value;
 }
 
-export function parse64I( sourceString: string ): number {
-  return parseInt(sourceString);
+export function parse64I( sourceString: string ): number | Long {
+  const long = Long.fromString(sourceString);
+  return {
+    upper: long.high,
+    lower: long.low
+  };
```

Comment:
I think we might need to use `isHexLiteral()` here to give ` Long.fromString` the correct radix argument?